src/PowerShellEditorServices.Host/EditorServicesHost.cs

```diff
@@ -303,6 +303,12 @@ private void OnDebugServiceClientConnect(object sender, TcpSocketServerChannel s
 
                         this.debugServiceListener.Start();
                     }
+                    else if (this.debugAdapter.IsUsingTempIntegratedConsole)

```

Comment:
So this else if prevents the host from exiting... What's the host in this case?

More simply, what is the user experience change?

Reply:
The temp debug console does not immediately exit after the script finishes execution or the debug session is stopped.  This allows the user to scroll through the window to inspect the script output.  

The only downside I've observed at this point is that every temp debug session log file ends with:
```
2/19/2018 8:48:16 PM [ERROR] - Method "ListenForMessages" at line 344 of C:\Users\Keith\GitHub\rkeithhill\PowerShellEditorServices\src\PowerShellEditorServices.Protocol\MessageProtocol\ProtocolEndpoint.cs

    Stream terminated unexpectedly, ending MessageDispatcher loop
    
    Exception: IOException
    Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host.
```
All things considered, I think this is a fine trade-off.

Reply:
100% agree!